api/src/main/java/org/hawkular/apm/api/utils/PropertyUtil.java

```diff
@@ -34,6 +34,21 @@
     public static final String HAWKULAR_TENANT = "HAWKULAR_TENANT";
 
     /**
+     * Location of the Hawkular Services Server.
+     */
+    public static final String HAWKULAR_URI = "HAWKULAR_URI";
+
+    /**
+     * Username for authentication against the Hawkular Server
+     */
+    public static final String HAWKULAR_USERNAME = "HAWKULAR_USERNAME";
+
+    /**
+     * Password for authentication against the Hawkular Server
```

Comment:
Can we use the same name for hawkular server? Either `Hawkular Server` or `Hawkular Services` I prefer the second one.